components/bitcoin-topics/layouts/TopicLayout.tsx

```diff
@@ -82,14 +55,41 @@ export default function TopicLayout({
         </a>
     )
 
+    useEffect(() => {
+        if (isNavOpen) {
```

Comment:
We're using similar piece of code in the `Topics Layout` and `Topics Banner` can this be abstracted into reusable functions and use in both places ? If it's not possible or will be time consuming please ignore for now